tests/test_champion.py

```diff
@@ -69,6 +69,11 @@ def test_ahri_stat_perlevel():
 def test_get_stats():
     annie = Annie(level=18)
     stats_annie = annie.__dict__
+    del stats_annie["passive"]
+    del stats_annie["q"]
+    del stats_annie["w"]
+    del stats_annie["e"]
+    del stats_annie["r"]
     stats_annie = {stat_name: round(stat, 2) for stat_name, stat in stats_annie.items()}
```

Comment:
To avoid changing this every time a new field is added in `BaseChamption`, it's probably better to just have individual asserts for each stat.

```
assert annie.level == 10
assert annie.hp == ...
...
```